app/controllers/concerns/authenticated_state_file_intake_concern.rb

```diff
@@ -5,13 +5,26 @@ module AuthenticatedStateFileIntakeConcern
     before_action :require_state_file_intake_login
   end
 
+  def current_intake
+    # NOTE: This intake is different from the unauthenticated value, stored in the regular session
+    # (Which the unauthenticated sessions controller uses)
+    unless @intake
+      current_intake_name = "current_#{question_navigator.intake_class.name.underscore}"
+      @intake = self.send(current_intake_name)
+    end
+    @intake
```

Comment:
I'm uncertain about the memoization. I like the tiny performance gain but I am unsure if it introduces risks. Either way, I think the explict authenticated intake method would be helpful here.

```suggestion
    @_authenticated_intake ||= current_authenticated_intake
```